src/panels/lovelace/custom-card-helpers.ts

```diff
@@ -3,3 +3,9 @@ export { createCardElement } from "./create-element/create-card-element";
 export { createBadgeElement } from "./create-element/create-badge-element";
 export { createHeaderFooterElement } from "./create-element/create-header-footer-element";
 export { createHuiElement } from "./create-element/create-hui-element";
+export { actionHandlerBind } from "./common/directives/action-handler-directive";
+export { handleAction } from "./common/handle-action";
+
+export const loadElements = {
+  entityEditor: () => import("./components/hui-entity-editor"),
```

Comment:
One last comment. Should the key here be the tag name? Or are we Camel casing it